src/components/GoalsList.tsx

```diff
@@ -0,0 +1,16 @@
+import type {Goal} from '../pages/Goals';
+
+export function GoalsList({goals = []}: {goals?: Goal[]}) {
+  return (
+    <>
+      {goals.map((goal) => (
+        <div style={{width: '100%', marginBottom: '20px'}} key={goal.RedditPostTitle}>
```

Comment:
nit: inline styling

Reply:
The styles were already like this so its all good. We can clean them up in another PR.